WebRoot/css/zm.css

```diff
@@ -4021,23 +4021,69 @@ HTML>BODY .appt-selected .appt_allday_body
 }
 
 /* ZmChooseFolderDialog and ZmPickTagDialog */
+
+#ChooseFolderDialog {
+	@ChooseFolderDialog@
+}
+
+#ChooseFolderDialog .WindowOuterContainer {
+	@ChooseFolderDialogContainer@
+}
+
 .ChooserDialog {
-	width:320px;
+	@ChooseFolderDialogBody@
```

Comment:
Whether it is contact or account, it signifies a kind of folder.
And, the property name is generic in that sense.
The styling will work for other dialogs too if not overrided.